### PR TITLE
Add UG for clear and delete command

### PR DESCRIPTION
**Pull request by WilliamRyank**

### CODE REVIEW

File: docs/UserGuide.adoc [hunk 1]
```diff
@@ -545,6 +545,23 @@ Example:
 * `done icP/S7896432A icD/S4578902G res/fail`
 // end::mark[]
 
+// tag::delete[]
+=== Deleting a person : `delete`
+
+If you want to delete a person in ORGANice you can type in a command in the following format: `delete **NRIC**`  +
+ORGANice will prevent you to delete a person if the person you want to delete has the following criteria:
```
Comment (TheRealRavager) [inline docs/UserGuide.adoc]: ORGANice will prevent you from deleting persons in the following categories:

File: docs/UserGuide.adoc [hunk 2]
```diff
@@ -545,6 +545,23 @@ Example:
...
+
+* It is a patient and his status is currently under processing.
+* It is a donor and his status is currently under processing.
+* It is a doctor and he is in charge of some number of patients in ORGANice.
```
Comment (TheRealRavager) [inline docs/UserGuide.adoc]: They are a ... and their status ...

In this case "they" is singular and gender neutral

File: docs/UserGuide.adoc [hunk 3]
```diff
@@ -545,6 +545,23 @@ Example:
...
+
+If you want to delete a patient that has NRIC numbver of S4612296H and is not currently under processing, type the following command: `delete S4612296H`
```
Comment (TheRealRavager) [inline docs/UserGuide.adoc]: numbver

File: docs/UserGuide.adoc [hunk 4]
```diff
@@ -545,6 +545,23 @@ Example:
...
+
+// end::delete[]
+=== Clearing all entries : `clear`
+
+You can clear all the entries in ORGANice if you found that the entries are long outdated by typing `clear`. +
```
Comment (TheRealRavager) [inline docs/UserGuide.adoc]: You can clear all entries in ORGANice by typing `clear`. You should run `clear` to start a fresh version of ORGANice after you have familiarised yourself with the commands using our dummy data.

File: docs/UserGuide.adoc [hunk 5]
```diff
@@ -549,18 +549,18 @@ Example:
...
+* They are a doctor and they are in charge of some number of patients in ORGANice.
 
-If you want to delete a patient that has NRIC numbver of S4612296H and is not currently under processing, type the following command: `delete S4612296H`
+If you want to delete a patient that has NRIC number of S4612296H and is not currently under processing, type the following command: `delete S4612296H`
 
 // end::delete[]
 === Clearing all entries : `clear`
 
-You can clear all the entries in ORGANice if you found that the entries are long outdated by typing `clear`. +
+You can clear all entries in ORGANice by typing clear. You should run clear to start a fresh version of ORGANice after you have familiarised yourself with the commands using our dummy data.
```
Comment (TheRealRavager) [inline docs/UserGuide.adoc]: `clear` instead of clear

File: docs/UserGuide.adoc [hunk 6]
```diff
@@ -545,6 +545,23 @@ Example:
 * `done icP/S7896432A icD/S4578902G res/fail`
 // end::mark[]
 
+// tag::delete[]
+=== Deleting a person : `delete`
+
+If you want to delete a person in ORGANice you can type in a command in the following format: `delete **NRIC**`  +
```
Comment (C-likethis123) [inline docs/UserGuide.adoc]: If you want to delete a person in ORGANice, you can type in a command of the following format:

File: docs/UserGuide.adoc [hunk 7]
```diff
@@ -545,6 +545,23 @@ Example:
...
+ORGANice will prevent you from deleting persons in the following categories:
```
Comment (C-likethis123) [inline docs/UserGuide.adoc]: ORGANice will prevent you from deleting a person if he/she is any of the following:

* A patient whose status is currently under processing.
* A donor whose status is currently under processing.
* A doctor who is in charge of one or more patients in ORGANice.

File: docs/UserGuide.adoc [hunk 8]
```diff
@@ -545,6 +545,23 @@ Example:
...
+
+* They are a patient and their status is currently under processing.
+* They are a donor and their status is currently under processing.
+* They are a doctor and they are in charge of some number of patients in ORGANice.
+
+If you want to delete a patient that has NRIC of S4612296H and is not currently under processing, type the following command: `delete S4612296H`
```
Comment (C-likethis123) [inline docs/UserGuide.adoc]: Here is an example. If you want to delete a patient with the NRIC S4612296H and is not under processing, type the following command: `delete S4612296H`.

File: docs/UserGuide.adoc [hunk 9]
```diff
@@ -545,6 +545,23 @@ Example:
...
+
+// end::delete[]
+=== Clearing all entries : `clear`
+
+You can clear all entries in ORGANice by typing `clear`. You should run clear to start a fresh version of ORGANice after you have familiarised yourself with the commands using our dummy data.
```
Comment (C-likethis123) [inline docs/UserGuide.adoc]: You can clear all data in ORGANice by typing `clear`. You could use this command to remove all dummy data after you have familiarized yourself with how ORGANice works.

**Pull request merged**